### PR TITLE
Independent Sampler always create a sample size of 1 no matter how small the scale

### PR DESCRIPTION
**Pull request by lajohn4747**
CU-86b0regxw
resolves [issue](https://github.com/datacebo/SDV-Enterprise/issues/607)

Prevents the too small of a sample from erroring out. Similar to: https://github.com/sdv-dev/SDV/issues/2045

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86b0regxw SDV-Enterprise - HSA Synthesizer's `scale` parameter doesn't work for small values #607](https://app.clickup.com/t/86b0regxw)

**Review by amontanez24 — COMMENTED** [review]
Looks good! Can we just add an integration test that basically copies the code for recreation in the issue?

**Comment by lajohn4747** [comment]
> Looks good! Can we just add an integration test that basically copies the code for recreation in the issue?

They are added in a different repo [here](https://github.com/datacebo/SDV-Enterprise/pull/657).

**Review by amontanez24 — APPROVED** [review]

File: tests/unit/sampling/test_independent_sampler.py [hunk 1]
```diff
@@ -404,3 +405,36 @@ def _connect_tables(sampled_data):
             'transactions': DataFrameMatcher(connected_transactions),
         })
         assert result == instance._finalize.return_value
+
+    def test__sample_too_small(self):
```
Comment (fealho) [inline tests/unit/sampling/test_independent_sampler.py:409]: Docstring

**Review by fealho — APPROVED** [review]
Looks good after adding an integration test.

**Pull request merged**